deployments/goerli/usdc/roots.json

```diff
@@ -1,12 +1,12 @@
 {
     "COMP": "0x3587b2F7E0E2D6166d6C14230e7Fe160252B0ba4",
-    "USDC": "0x8bF5b658bdF0388E8b482ED51B14aef58f90abfD",
+    "USDC": "0x07865c6E87B9F70255377e024ace6630C1Eaa37F",
```

Comment:
I think you'll also need to add `baseTokenAddress: "0x07865c6E87B9F70255377e024ace6630C1Eaa37F"` to `configuration.json` now that we aren't cloning USDC.

Reply:
Hmm, this should mean current deployment is broken in a weird way, wonder if Treasury will hit it before I redeploy...

Reply:
Actually, no, its just referencing by name as it should be https://github.com/compound-finance/comet/blob/main/deployments/goerli/usdc/configuration.json#L4

Reply:
Right, but referencing by name shouldn't work anymore because we aren't cloning `USDC`, so there's no contract address that maps to `USDC` in the cache. That's why I think we'll need to add `baseTokenAddress` to the configuration. Does the deploy script actually work if you wipe out your cache?

Reply:
That's why it's a root?

Reply:
Ugh, roots should be aliases, good point.

Reply:
Ok, should be fixed in the next 2 commits (patch and then re-deploy), looks right on etherscan: https://goerli.etherscan.io/address/0x3EE77595A8459e93C2888b13aDB354017B198188#readProxyContract

Reply:
So I see you addressed the roots/aliases issue, but that's actually not the suggestion I was making. We have the ability to specify token addresses directly in `configuration.json` and I think that's where these hardcoded addresses should live. IMO, deployment scripts shouldn't require there to be pre-existing roots to work. A developer should be able to run a deployment script with a `configuration.json` file and have it just work, creating a `roots.json` in the process. Let's maybe sync on this tmr during parking lot?